lib/verk/workers_manager.ex

```diff
@@ -193,6 +193,8 @@ defmodule Verk.WorkersManager do
         Log.start(job, worker)
         Verk.Worker.perform_async(worker, self, job)
         notify!(%Events.JobStarted{job: job, started_at: Timex.DateTime.now})
+      :full ->
+        Verk.enqueue(job)
```

Comment:
I think so. We should always get the amount of workers we know that poolboy has. It should never be less than the amount we just asked for I guess.